langwatch/src/components/MainMenu.tsx

```diff
@@ -66,8 +58,8 @@ export const MainMenu = React.memo(
         borderRightColor="gray.300"
         background="white"
         transition="all 0.2s ease-in-out"
-        width={isExpanded ? "200px" : menuWidth + "px"}
-        minWidth={isExpanded ? "200px" : menuWidth + "px"}
+        width={menuWidth + "px"}
+        minWidth={menuWidth + "px"}
```

Comment:
Consider defining a single variable above and then using that. Ideally, we should avoid writing code inline in props.